package.json

```diff
@@ -120,6 +121,7 @@
     "redux-mock-store": "1.5.4",
     "redux-saga-test-plan": "4.0.5",
     "sass": "1.53.0",
+    "source-map-explorer": "2.5.2",
```

Comment:
Do you think we should run this via npx rather than maintaining this and all its deps? It's probably not something we're going to run regularly.

Reply:
I was actually thinking we might want to run that regularly and keep monitoring the bundle size continuously. Good suggestion - will replace with `npx` for now.